client/core/trade.go

```diff
@@ -1329,10 +1402,41 @@ func (c *Core) redeemMatches(t *trackedTrade, matches []*matchTracker) error {
 	// If an error was encountered, fail all of the matches. A failed match will
 	// not run again on during ticks.
 	if err != nil {
+		// The caller will notify the user that there is a problem. We really
+		// have no way of knowing whether this is recoverable (so we can't set
+		// failErr), but we do want to prevent redemptions every tick.
 		for _, match := range matches {
-			match.failErr = err
+			// Mark these matches as suspect. Suspect matches will not be
+			// grouped for redemptions in future attempts.
+			match.suspectRedeem = true
+			// If we can still make a broadcast timeout, allow retries soon. It
+			// is possible that RedemptionStamp or AuditStamp is be zero if
+			// we're recovering during startup. The implications of this is that
+			// if we are 1) already in start up recovery, and 2) we fail again,
+			// we will not try again for an hour even if we have time before the
+			// broadcast timeout.
+			lastActionStamp := match.MetaData.Proof.Auth.AuditStamp
+			if match.Match.Side == order.Taker {
+				lastActionStamp = match.MetaData.Proof.Auth.RedemptionStamp
+			}
+			lastActionTime := encode.UnixTimeMilli(int64(lastActionStamp))
+			// Try to wait until about the next auto-tick to try again.
+			meterDuration := t.dc.tickInterval * 3 / 4
+			if time.Since(lastActionTime) > t.broadcastTimeout() {
+				// If we already missed the broadcast timeout, we're not in as
+				// much of a hurry. but keep trying and sending errors, because
+				// we do want the user to recover.
+				meterDuration = time.Hour
```

Comment:
I'd consider ~5 min.  Considerations for making it 1hr?

Reply:
It's all about the log spamming. The user has already been informed multiple times that there is a problem. If they take corrective action by reconfiguring their wallet, or if they restart dexc, the tick meterer is cleared. But there are probably other corrective actions that wouldn't.

I can go to 5 minutes to start. I'm just anticipating someone being jarred by their 99+ error notifications.

Reply:
Yeah, it's a balance between log spamming and sweating for an hour.